### DISCUSSION TITLE
q-select options object property with array of objects inside it

### DISCUSSION THREAD

**Discussion by numero14-dev** [quasarframework/quasar#9158] [General - Components / Directives / etc] [answered]
Hi
I'm trying to show my options data list using the q-select component. but my option list is a little different. I have a property (List) with an array of objects inside the options array of objects.

My options like this: 

`      options: [
        {
          label: 'Users',
          list: [
            { name: 'John', lastName: 'Doe' },
            { name: 'Martin', lastName: 'Devae' },
          ],
        },
        {
          label: 'Admins',
          list: [
            { name: 'Mark', lastName: 'Zenbio' },
            { name: 'Povel', lastName: 'Kole' },
          ],
        },
      ],
`

and my code:

`<q-select
                  filled
                  v-model="model"
                  :options="options"
                  dense
                  options-selected-class="text-cyan-7"
                >
                  <template v-slot:option="scope">
                    <div class="labels" v-bind="scope.itemProps" v-on="scope.itemEvents">{{ scope.opt.label }}</div>
                    <q-item v-bind="scope.itemProps" v-on="scope.itemEvents">
                      <q-item-section v-bind="scope.itemProps" v-on="scope.itemEvents">
                        <q-item-label v-html="scope.opt.list" />
                      </q-item-section>
                    </q-item>
                  </template>
                  <template v-slot:selected> {{ model.list }} </template>
                </q-select>`

now I don't know how I can have another loop inside the 'list' property to show Something like this image I attached.

labels are not selectable.

![Qselect](https://user-images.githubusercontent.com/72903442/116968125-1bd0ba00-acc9-11eb-982d-d82959ff8516.png)

Thanks

**Comment by metalsadman** [comment, marked as answer]
@numero14-dev something like this https://codepen.io/metalsadman/pen/QWpLjKa?editors=1000.

**Reply by numero14-dev** [reply]
Thank you!
